src/ft/mod.rs

```diff
@@ -105,6 +105,27 @@ fn to_fixedpoint_16_6(f: f64) -> c_long {
     (f * 65536.0) as c_long
 }
 
+#[inline]
+fn from_freetype_26_6(f: impl IntoF32) -> f32 {
+    f.into_f32() / 64.
+}
+
+trait IntoF32 {
+    fn into_f32(self) -> f32;
+}
+
+impl IntoF32 for f32 {
+    fn into_f32(self) -> f32 {
+        self
+    }
+}
+
+impl IntoF32 for i64 {
+    fn into_f32(self) -> f32 {
+        self as f32
+    }
+}
```

Comment:
This trait is necessary because `i64` doesn't implement `Into<f32>` since it's not a lossless conversion.

Reply:
Yeah, that sounds fine.